<launch>
    <include file="$(find realsense2_camera)/launch/rs_camera.launch">
        <arg name="enable_gyro" value="false"/> <arg name="enable_accel" value="false"/>
        
        <arg name="enable_infra1" value="true"/>
        <arg name="enable_infra2" value="true"/>
        <arg name="enable_color" value="false"/>
        <arg name="enable_depth" value="false"/>

        <arg name="enable_sync" value="true"/>
        
        <arg name="infra_width" value="848"/>
        <arg name="infra_height" value="480"/>
        <arg name="infra_fps" value="30"/>
    </include>
</launch>